includes/admin/class-wc-admin-notices.php

```diff
@@ -8,6 +8,14 @@
 
 defined( 'ABSPATH' ) || exit;
 
+/**
+ * Name for notice for showing minimum version requirements. Change value to invalidate previous dismissals when requirement changes in future.
+ * Last changed in 3.8.
+ *
+ * @since 3.8.0.
+ */
+define( 'WC_PHP_MIN_REQUIREMENTS_NOTICE', 'wp_php_min_requirements_3_8' );
```

Comment:
While nothing is wrong with this we just need to be mindful that this will be yet another thing to remember to update when we introduce new minimums with each release.